src/pages/builders/materials.tsx

```diff
@@ -7,18 +9,130 @@ import type { Config } from '@/types/config.type'
 
 export type MaterialsProps = Pick<Config, 'title' | 'keywords' | 'description' | 'urls'>
 
+type MaterialLink = {
+  name: string
+  label: string
+  url: string
+}
+
 const Materials: NextPage<MaterialsProps> = props => {
   const { urls } = props
+  const {
+    materialsUrls: {
+      whitePaperUrl,
+      technicalDocsUrl,
+      tutorialVideosUrl,
+      githubOkp4dUrl,
+      githubContractsUrl,
+      githubAwesomeUrl,
+      discordChannelUrl,
+      discordTicketChannelUrl
+    },
+    socialMediaUrls: { mediumUrl }
+  } = urls
+
+  const materialsLinks: MaterialLink[] = useMemo(
+    () => [
+      {
+        name: 'Whitepaper',
+        label: 'whitepaper',
+        url: whitePaperUrl
+      },
+      {
+        name: 'Technical documentation',
+        label: 'technical-docs',
+        url: technicalDocsUrl
+      },
+      {
+        name: 'Tutorial videos',
```

Comment:
If this one has no available link for now, please do not reload page with a `#` route redirection but disable the button behavior instead